src/components/dropdownSearch/examples/DropdownSearchExamples.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { DropdownSearchConnected } from '../DropdownSearchConnected';
+import { ReduxConnect } from '../../../utils/ReduxUtils';
+import * as _ from 'underscore';
+import { UUID } from '../../../utils/UUID';
+
+const mapStateToProps = () => {
+  return {};
+};
+
+const mapDispatchToProps = () => {
+  return {};
+};
+
+@ReduxConnect(mapStateToProps, mapDispatchToProps)
```

Comment:
I do not think you need to use Redux  connect here since you do not have anything to connect.